src/styles/blocks/card.scss

```
@@ -0,0 +1,80 @@
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  width: $card-width;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    line-height: 18px;
+
+    font-size: $font-sz-normal;
+    font-weight: $font-w-normal;
+    color: $color-dark;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-size: $font-sz-small;
+    color: $grey-color;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    color: $grey-color;
+    font-size: $font-sz-normal;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: $font-sz-large;
+    font-weight: $font-w-bold;
+    line-height: 18px;
+    color: $color-dark;
+  }
+
+  &__buy-button {
+    display: block;
+    height: 40px;
+
+    font-size: $font-sz-medium;
+    text-transform: uppercase;
+    font-weight: $font-w-bold;
+    color: $regular-color;
+    background-color: $border-blue-color;
+
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    line-height: 40px;
+    text-align: center;
+    transition: 0.3s ease-in-out;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```